packages/image_picker/image_picker_windows/lib/image_picker_windows.dart

```diff
@@ -164,4 +164,29 @@ class ImagePickerWindows extends ImagePickerPlatform {
         .openFiles(acceptedTypeGroups: <XTypeGroup>[typeGroup]);
     return files;
   }
+
+  // `maxWidth`, `maxHeight`, and `imageQuality` arguments are not
+  // supported on Windows. If any of these arguments is supplied,
+  // it'll be silently ignored by the Windows version of the plugin.
```

Comment:
"they will be"

Remove "by the Windows version of the plugin"

(I fixed these issues in the existing comments in a pending PR, so no need to change them there.)